ocp_resources/utils.py

```diff
@@ -276,3 +277,14 @@ def _return_resource(_resource, _check_exists, _msg):
                     _check_exists=check_exists,
                     _msg=skip_create_warn_msg,
                 )
+
+
+def wait_status_not_exist(vm, status, timeout):
```

Comment:
status may exist but be None or False:
1. is it ok to exit if it is False?
2. Please rename